src/m365/spo/commands/tenant/tenant-recyclebinitem-remove.spec.ts

```diff
@@ -100,27 +100,27 @@ describe(commands.TENANT_RECYCLEBINITEM_REMOVE, () => {
 
     const pastDate = new Date();
     pastDate.setSeconds(pastDate.getSeconds() - 1800);
-    sinon.stub(spo, 'ensureFormDigest').callsFake(() => { return Promise.resolve({ FormDigestValue: 'abc', FormDigestTimeoutSeconds: 1800, FormDigestExpiresAt: pastDate, WebFullUrl: 'https://contoso.sharepoint.com/sites/hr' }); });
+    sinon.stub(spo, 'ensureFormDigest').resolves({ FormDigestValue: 'abc', FormDigestTimeoutSeconds: 1800, FormDigestExpiresAt: pastDate, WebFullUrl: 'https://contoso.sharepoint.com/sites/hr' });
```

Comment:
These items can be removed. 

also: don't forget the afterEach stub restoring spo.ensureFormDigest, that will have to go as well.